crates/solidity/testing/snapshots/bindings_output/built_ins/address/generated/0.5.0-success.txt

```diff
@@ -65,3 +65,42 @@ References and definitions:
    │                                        │     
    │                                        ╰───── ref: built-in
 ───╯
+Definiens: 
```

Comment:
I think this is great as is, but I wonder why was it split into a new graph? was it too noisy to add to the same one?

Reply:
Yes, I found the output confusing, especially because most definiens will include the leading trivia. So that quickly generates a lot of lines in the render and it makes is less readable.